src/main/java/de/retest/ui/PathElement.java

```diff
@@ -85,11 +77,7 @@ public boolean equals( final Object obj ) {
 		} else if ( !elementName.equals( other.elementName ) ) {
 			return false;
 		}
-		if ( suffix == null ) {
-			if ( other.suffix != null ) {
-				return false;
-			}
-		} else if ( !suffix.equals( other.suffix ) ) {
+		if ( !(suffix == other.suffix) ) {
```

Comment:
We can probably remove the lines `if ( elementName == null ) ...` because it looks like the field can never be `null`. What do you think?